### ISSUE TITLE
"Online Validator" runs for 5 minutes and displays no results

### ISSUE THREAD

**Issue by StefanRohde-Enslin** [EasyinnovaSL/DPFManager#252]
Just called http://dpfmanager.org/application.html and uploaded a file. Choose configuration Baseline xml.dpf and waited for 5 minutes after click on "check file". A "current task" ... sending ... appeared and after 5 minutes disappeared. That's all that happend.

**Comment by victormunoz** [comment]
Thank you, we will check and fix it as soon as possible.

**Comment by victormunoz** [comment]
Hi, Stefan, we have found a problem when sending a non-Tiff file to validate. In this case the task starts but never ends. We are going to fix it. Which file did you try to check?

**Comment by StefanRohde-Enslin** [comment]
Hi Victor,

I tried to mai the file I used for testing the online validator. It was too big. So I uploaded it to: http://www.museum-digital.de/ausgangskarte.tif
Please inform me, when you downloaded it – so I can keep my server clean.

Best
Stefan


Von: Victor Muñoz [mailto:notifications@github.com]
Gesendet: Dienstag, 30. August 2016 08:14
An: EasyinnovaSL/DPFManager
Cc: Rohde-Enslin, Stefan; Author
Betreff: Re: [EasyinnovaSL/DPFManager] "Online Validator" runs for 5 minutes and displays no results (#252)


Hi, Stefan, we have found a problem when sending a non-Tiff file to validate. In this case the task starts but never ends. We are going to fix it. Which file did you try to check?

**Comment by victormunoz** [comment]
I have downloaded the file, checked it with the online validator, and the file has been succesfully validated (in about 2 seconds).

Can you try to check it again? If it fails again, please tell me the browser that you are using, so that I can reproduce the problem

**Comment by StefanRohde-Enslin** [comment]
Hi Victor,

uploaded the Tiff-File, choose again configuration=baselinexml.dpf and waited for 3 minutes. Then the “current tasks” – line disappeared and nothing else happened. Except that this time for a very short moment a message appeared right below the configuration-choosing-line saying: “ERROR: Server error”. After about a second this message disappeared.
The browser I used was a newly installed firefox 48.0.
Than I tried with IE 11.0 and got the same Error message which disappeared again after one second.

Best
Stefan

Von: Victor Muñoz [mailto:notifications@github.com]
Gesendet: Dienstag, 30. August 2016 10:24
An: EasyinnovaSL/DPFManager
Cc: Rohde-Enslin, Stefan; Author
Betreff: Re: [EasyinnovaSL/DPFManager] "Online Validator" runs for 5 minutes and displays no results (#252)


I have downloaded the file, checked it with the online validator, and the file has been succesfully validated (in about 2 seconds).

Can you try to check it again? If it fails again, please tell me the browser that you are using, so that I can reproduce the problem

**Comment by victormunoz** [comment]
I have been able to reproduce the issue. I will notify you when it's been solved. Thanks for reporting!

**Comment by victormunoz** [comment]
This issue has been solved!

**Comment by StefanRohde-Enslin** [comment]
Hi Victor,

the issue 252 of the dpf-manager (online validator …) is not solved. Still everything is as before.

Best
Stefan

Von: Victor Muñoz [mailto:notifications@github.com]
Gesendet: Donnerstag, 1. September 2016 12:45
An: EasyinnovaSL/DPFManager
Cc: Rohde-Enslin, Stefan; Author
Betreff: Re: [EasyinnovaSL/DPFManager] "Online Validator" runs for 5 minutes and displays no results (#252)


This issue has been solved!

**Comment by StefanRohde-Enslin** [comment]
Checked if the issue is solved in same setting as before (ff48 and ie11) with same file: Same result = no result / report

**Comment by victormunoz** [comment]
Yes but now the message "Requested entity too large" should appear, because there is a limit of 5Mb in the images that are uploaded to be checked by the online validator.

**Comment by StefanRohde-Enslin** [comment]
Well I started the "sending"-process and than was away from the computer for some minutes. When I came back there was no message. It might be possible that a "entity too large" message appeared and disappeared after a short time. It was not there when I looked again at the screen.
If there is limit of 5Mb it should be stated clearly. For example one could write "File (5Mb max.)" instead of "Files" (where I can upload only one File).
